src/calculate.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const calculateCowsAndBulls = (number, answer) => {
+  return answer.split('').reduce((count, char, index) => {
+    if (char === number[index]) {
+      return {
+        ...count,
+        bulls: count.bulls + 1,
+      };
+    }
+
+    if (number.includes(char)) {
+      return {
+        ...count,
+        cows: count.cows + 1,
+      };
+    }
+
+    return count;
```

Comment:
```suggestion
    count.bulls += char === number[index];
    count.cows += (
      char !== number[index] && number.includes(char)
    );

    return count;
```

Use the fact that boolean will be converted to a number when using plus.